benchmarks/ExecutionContextBenchmarks.cpp

```diff
@@ -0,0 +1,268 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of clio: https://github.com/XRPLF/clio
+    Copyright (c) 2023, the clio developers.
```

Comment:
2024

Reply:
This file was added in 2023 😃 I will update 👍